pkg/apis/externaldns/types.go

```diff
@@ -105,6 +107,7 @@ func (cfg *Config) ParseFlags(args []string) error {
 	app.Flag("domain-filter", "Limit possible target zones by a domain suffix; specify multiple times for multiple domains (optional)").Default("").StringsVar(&cfg.DomainFilter)
 	app.Flag("azure-config-file", "When using the Azure provider, specify the Azure configuration file (required when --provider=azure").Default(defaultConfig.AzureConfigFile).StringVar(&cfg.AzureConfigFile)
 	app.Flag("azure-resource-group", "When using the Azure provider, override the Azure resource group to use (optional)").Default(defaultConfig.AzureResourceGroup).StringVar(&cfg.AzureResourceGroup)
+	app.Flag("cloudflare-proxied", "When using the Cloudflare provider, specify if records should receive the performance and security benefits of Cloudflare (default: disabled)").BoolVar(&cfg.CloudflareProxied)
```

Comment:
The description sounds a bit sales'ish ("performance and security benefits of Cloudflare") :)
Maybe just say it enables proxied mode.

Reply:
Description updated (it was copied from Cloudflare's API docs, but indeed it's sales'ish).